app/assets/stylesheets/application.css.scss

```diff
@@ -8,21 +8,28 @@
  * You're free to add application-wide styles to this file and they'll appear at the top of the
  * compiled file, but it's generally better to create a new file per style scope.
  *
- *= require scaffolds
- *= require_self
- *= require home
- *= require loans
- *= require users
- *= require media
- *= require embedded
- *= require font-awesome
- *= require jasny-bootstrap
  */
 
 // Bootstrap v3.1.1 is already loaded in the WP theme
 //@import "bootstrap-sprockets";
 //@import "bootstrap";
 
+// Bootstrap Sass
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "jasny-bootstrap";
+
+// Vendor Styles
+@import "font-awesome";
+
+// Application Styles
+@import "scaffolds";
+@import "home";
+@import "loans";
+@import "users";
+@import "media";
+@import "embedded";
+
```

Comment:
`@import` directives in SCSS are also included one file, during the SCSS compilation phase, and it will also make SCSS variables & defined mixins and the suchlike available in any of the files imported afterwards. It's pretty standard practice when using SCSS-based layout frameworks, and I am pretty sure the next bootstrap is heading that direction too.